### ISSUE TITLE
Cannot read property 'toString' of undefined

### ISSUE THREAD

**Issue by owen-soak** [daun/laravel-mix-transpile-node-modules#6]
Currently experiencing this error

```
 TypeError: Cannot read property 'toString' of undefined
    at TranspileNodeModules.ruleTransformsJs (/Users/owen.soak/Sites/projectx/node_modules/laravel-mix-transpile-node-modules/index.js:69:26)
    at /Users/owen.soak/Sites/projectx/node_modules/laravel-mix-transpile-node-modules/index.js:64:22
    at Array.filter (<anonymous>)
    at TranspileNodeModules.findBabelRules (/Users/owen.soak/Sites/projectx/node_modules/laravel-mix-transpile-node-modules/index.js:63:18)
    at TranspileNodeModules.webpackConfig (/Users/owen.soak/Sites/projectx/node_modules/laravel-mix-transpile-node-modules/index.js:51:24)
    at /Users/owen.soak/Sites/projectx/node_modules/laravel-mix/src/components/ComponentRegistrar.js:124:54
    at /Users/owen.soak/Sites/projectx/node_modules/laravel-mix/src/Dispatcher.js:39:62
    at Array.map (<anonymous>)
    at Dispatcher.fire (/Users/owen.soak/Sites/projectx/node_modules/laravel-mix/src/Dispatcher.js:39:47)
    at Mix.dispatch (/Users/owen.soak/Sites/projectx/node_modules/laravel-mix/src/Mix.js:250:32)
npm ERR! code ELIFECYCLE
npm ERR! errno 2
```

**Comment by owen-soak** [comment]
> 

Sorry updated the above, looks like github cropped some of it off due to the code snippet starting with "[webpack-cli]"

**Comment by owen-soak** [comment]
> Does the error go away if you disable `laravel-mix-polyfill`? I can vaguely remember them not working together but haven't use polyfill myself.

Removed this and the same error crops up.

**Comment by daun** [comment]
I'm afraid I'm not able to look into this at the moment and won't be for a few more weeks. If you decide to dig deeper and find a fix, I'm happy to take a pull request.

**Comment by daun** [comment]
This should be fixed in `2.0.4`, please reopen the issue if you're still seeing it.
